doc/version-control-fields.rst

```diff
@@ -0,0 +1,179 @@
+Version Control System Fields
+=============================
+
+Most of the version control system (VCS) fields types are common to both
+``source-repository`` and ``source-repository-package`` stanzas.
```

Comment:
This `version-control-fields.rst` page feels out of place compared to the other reference pages:
The others summarize a high-level thing like the whole file formats or a major part of the CLI interface. This one picks and focuses on two minor stanzas, and lacks context (page can't be read on its own).
If we decide to keep this page (which i strongly discourage, but won't block if others are fine with it), it should have a context introductory sentence that links to the two other reference pages.

Reply:
This page now only has VCS fields. The `source-repository` is back with `doc/cabal-package-description-file.rst` and the `source-repository-package` is back with `doc/cabal-project-description-file.rst`.